test/integration/targets/git/vars/main.yml

```diff
@@ -15,10 +15,10 @@ separate_git_dir: '{{ output_dir }}/sep_git_dir'
 repo_format1: 'https://github.com/jimi-c/test_role'
 repo_format2: 'git@github.com:jimi-c/test_role.git'
 repo_format3: 'ssh://git@github.com/jimi-c/test_role.git'
-repo_submodules: 'https://github.com/abadger/test_submodules.git'
-repo_submodules_newer: 'https://github.com/abadger/test_submodules_newer.git'
-repo_submodule1: 'https://github.com/abadger/test_submodules_subm1.git'
-repo_submodule1_newer: 'https://github.com/abadger/test_submodules_subm1_newer.git'
+repo_submodules: 'https://github.com/ju2wheels/test_submodules.git'
```

Comment:
@ju2wheels is it possible to embed or better autogenerate those repos locally? For this test, protocol probably doesn't matter. `git clone ./local/path/to/bare/repo.git` would work as well.

Reply:
@ju2wheels otherwise, let's update @abadger's repo maybe?

Reply:
Resolved this by removing remote repos and using dynamically created local ones.